pkg/sql/schemachanger/scplan/testdata/drop_view

```diff
@@ -715,83 +667,43 @@ DROP VIEW defaultdb.v1 CASCADE
+- from: [ColumnComment:{DescID: 106, PgAttributeNum: 2}, ABSENT]
   to:   [View:{DescID: 106}, DROPPED]
   kind: Precedence
   rule: dependent element removal before descriptor drop
-- from: [ColumnComment:{DescID: 107, ColumnID: 1}, ABSENT]
-  to:   [Column:{DescID: 107, ColumnID: 1}, ABSENT]
-  kind: Precedence
-  rule: dependents removed before column
-- from: [ColumnComment:{DescID: 107, ColumnID: 1}, ABSENT]
+- from: [ColumnComment:{DescID: 107, PgAttributeNum: 1}, ABSENT]
   to:   [View:{DescID: 107}, DROPPED]
   kind: Precedence
   rule: dependent element removal before descriptor drop
-- from: [ColumnComment:{DescID: 107, ColumnID: 2}, ABSENT]
-  to:   [Column:{DescID: 107, ColumnID: 2}, ABSENT]
-  kind: Precedence
-  rule: dependents removed before column
-- from: [ColumnComment:{DescID: 107, ColumnID: 2}, ABSENT]
+- from: [ColumnComment:{DescID: 107, PgAttributeNum: 2}, ABSENT]
   to:   [View:{DescID: 107}, DROPPED]
   kind: Precedence
   rule: dependent element removal before descriptor drop
-- from: [ColumnComment:{DescID: 108, ColumnID: 1}, ABSENT]
-  to:   [Column:{DescID: 108, ColumnID: 1}, ABSENT]
-  kind: Precedence
-  rule: dependents removed before column
-- from: [ColumnComment:{DescID: 108, ColumnID: 1}, ABSENT]
+- from: [ColumnComment:{DescID: 108, PgAttributeNum: 1}, ABSENT]
   to:   [View:{DescID: 108}, DROPPED]
   kind: Precedence
   rule: dependent element removal before descriptor drop
-- from: [ColumnComment:{DescID: 108, ColumnID: 2}, ABSENT]
-  to:   [Column:{DescID: 108, ColumnID: 2}, ABSENT]
-  kind: Precedence
-  rule: dependents removed before column
-- from: [ColumnComment:{DescID: 108, ColumnID: 2}, ABSENT]
+- from: [ColumnComment:{DescID: 108, PgAttributeNum: 2}, ABSENT]
   to:   [View:{DescID: 108}, DROPPED]
   kind: Precedence
   rule: dependent element removal before descriptor drop
-- from: [ColumnComment:{DescID: 111, ColumnID: 1}, ABSENT]
-  to:   [Column:{DescID: 111, ColumnID: 1}, ABSENT]
-  kind: Precedence
-  rule: dependents removed before column
-- from: [ColumnComment:{DescID: 111, ColumnID: 1}, ABSENT]
+- from: [ColumnComment:{DescID: 111, PgAttributeNum: 1}, ABSENT]
   to:   [View:{DescID: 111}, DROPPED]
   kind: Precedence
   rule: dependent element removal before descriptor drop
-- from: [ColumnComment:{DescID: 111, ColumnID: 2}, ABSENT]
-  to:   [Column:{DescID: 111, ColumnID: 2}, ABSENT]
-  kind: Precedence
-  rule: dependents removed before column
-- from: [ColumnComment:{DescID: 111, ColumnID: 2}, ABSENT]
+- from: [ColumnComment:{DescID: 111, PgAttributeNum: 2}, ABSENT]
   to:   [View:{DescID: 111}, DROPPED]
   kind: Precedence
   rule: dependent element removal before descriptor drop
-- from: [ColumnComment:{DescID: 111, ColumnID: 3}, ABSENT]
-  to:   [Column:{DescID: 111, ColumnID: 3}, ABSENT]
-  kind: Precedence
-  rule: dependents removed before column
-- from: [ColumnComment:{DescID: 111, ColumnID: 3}, ABSENT]
+- from: [ColumnComment:{DescID: 111, PgAttributeNum: 3}, ABSENT]
```

Comment:
I am glad that these tests exist. This diff and others like it tell me that you forgot to update the rule definitions in the `rules` package to take into account that the column comment element now identifies columns by PGAttributeNum instead of ColumnID.